README.md

```diff
@@ -323,6 +323,41 @@ The documentation is rendered using an interactive terminal session that you can
 
 run `fastify docs` to get started.
 
+## Test helpers
+
+When you use `fastify-cli` to run your project you need a way to test your application.
+To do so, you can use the this module to load your application and give you the control to write your assertions.
+
+There are two utilities provided:
+
+- `build`: builds your application and returns the `fastify` instance without calling the `listen` method.
+- `listen`: starts your application and returns the `fastify` instance listening on the configured port.
+
+Both of these utilities have the `function(arg, pluginOptions)` parameters:
+
+- `arg`: is a string or a string array within the same arguments passed to the `fastify-cli` command.
```

Comment:
I would rename it to something more understandable, what o you think about `cliArgs` ?